tools/NuGet/template-artifactory/DynamoVisualProgramming.DynamoCoreRuntime.nuspec

```diff
@@ -1,18 +1,22 @@
 <?xml version="1.0"?>
 <package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
-  <metadata>
-    <id>DynamoVisualProgramming.DynamoCoreRuntime</id>
-    <version>$Version$</version>
-    <authors>Autodesk</authors>
-    <owners>Autodesk</owners>
-    <licenseUrl>https://www.apache.org/licenses/LICENSE-2.0</licenseUrl>
-    <projectUrl>https://github.com/DynamoDS/Dynamo</projectUrl>
-    <iconUrl>https://raw.githubusercontent.com/DynamoDS/Dynamo/master/doc/distrib/Images/logo_square_32x32.png</iconUrl>
-    <requireLicenseAcceptance>false</requireLicenseAcceptance>
-    <description>This package bundles all files from the Dynamo bin folder except for test libraries and symbol files.</description>
-    <copyright>Copyright Autodesk 2019</copyright>
```

Comment:
I guess we can also update these to 2020?

Reply:
Yes, I'll do the change

Reply:
Change done